mindee/documents/passport/passport_v1.py

```diff
@@ -67,25 +67,55 @@ def _build_from_api_prediction(
         self, api_prediction: TypeApiPrediction, page_n: Optional[int] = None
     ) -> None:
         """
-        Build the document from an API response JSON.
+        Build the object from the prediction API JSON.
 
         :param api_prediction: Raw prediction from HTTP response
         :param page_n: Page number for multi pages pdf input
         """
-        self.country = TextField(api_prediction["country"], page_id=page_n)
-        self.id_number = TextField(api_prediction["id_number"], page_id=page_n)
-        self.birth_date = DateField(api_prediction["birth_date"], page_id=page_n)
-        self.expiry_date = DateField(api_prediction["expiry_date"], page_id=page_n)
-        self.issuance_date = DateField(api_prediction["issuance_date"], page_id=page_n)
-        self.birth_place = TextField(api_prediction["birth_place"], page_id=page_n)
-        self.gender = TextField(api_prediction["gender"], page_id=page_n)
-        self.surname = TextField(api_prediction["surname"], page_id=page_n)
-        self.mrz1 = TextField(api_prediction["mrz1"], page_id=page_n)
-        self.mrz2 = TextField(api_prediction["mrz2"], page_id=page_n)
+        self.birth_date = DateField(
+            api_prediction["birth_date"],
+            page_id=page_n,
+        )
+        self.birth_place = TextField(
+            api_prediction["birth_place"],
+            page_id=page_n,
+        )
+        self.country = TextField(
+            api_prediction["country"],
+            page_id=page_n,
+        )
+        self.expiry_date = DateField(
+            api_prediction["expiry_date"],
+            page_id=page_n,
+        )
+        self.gender = TextField(
+            api_prediction["gender"],
+            page_id=page_n,
+        )
         self.given_names = [
-            TextField(given_name, page_id=page_n)
-            for given_name in api_prediction["given_names"]
+            TextField(prediction, page_id=page_n)
+            for prediction in api_prediction["given_names"]
         ]
+        self.id_number = TextField(
+            api_prediction["id_number"],
+            page_id=page_n,
+        )
+        self.issuance_date = DateField(
+            api_prediction["issuance_date"],
+            page_id=page_n,
+        )
+        self.mrz1 = TextField(
+            api_prediction["mrz1"],
+            page_id=page_n,
+        )
+        self.mrz2 = TextField(
+            api_prediction["mrz2"],
+            page_id=page_n,
+        )
+        self.surname = TextField(
+            api_prediction["surname"],
+            page_id=page_n,
+        )
         self.mrz = TextField({"value": None, "confidence": 0.0}, page_id=page_n)
```

Comment:
Is this still relevant? I believe we rolled it out in other SDKs

Reply:
This will be removed in v4, but for now we need to keep retro compatibility.